### PR TITLE
Bug/ad 38 ideal refusal result

### PR DESCRIPTION
**Pull request by er1z**

### CODE REVIEW

File: src/Processor/PaymentResponseProcessor.php [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    private $processors;
+
+    /** @var UrlGeneratorInterface */
+    private $urlGenerator;
+
+    /**
+     * PaymentResponseProcessor constructor.
+     *
+     * @param Processor[] $processors
+     */
```
Comment (patrick477) [inline src/Processor/PaymentResponseProcessor.php]: redundant

File: src/Processor/PaymentResponseProcessor.php [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+use Webmozart\Assert\Assert;
+
+final class PaymentResponseProcessor implements PaymentResponseProcessorInterface
+{
+    private const DEFAULT_REDIRECT_ROUTE = 'sylius_shop_order_thank_you';
+
+    /** @var Processor[] */
```
Comment (patrick477) [inline src/Processor/PaymentResponseProcessor.php]: interface?

File: src/Resolver/Payment/PaymentDetailsResolverInterface.php [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+/*
+ * This file has been created by developers from BitBag.
+ * Feel free to contact us once you face any issues or want to start
+ * You can find more information about us on https://bitbag.io and write us
+ * an email on hello@bitbag.io.
+ */
+
+declare(strict_types=1);
+
+
```
Comment (patrick477) [inline src/Resolver/Payment/PaymentDetailsResolverInterface.php]: empty line

**Review by patrick477 — APPROVED** [review]

File: src/Processor/PaymentResponseProcessor.php (hunk 1 above)
Comment (er1z) [reply src/Processor/PaymentResponseProcessor.php]: Doesn't pass the Psalm without this docblock.

**Review by patrick477 — APPROVED** [review]

**Pull request merged**